Sources/Task/TaskRecover.swift

```diff
@@ -47,4 +47,18 @@ extension Task {
         return Task<SuccessValue>(future: future, cancellation: cancel)
         #endif
     }
+
+    public func recoverWithNewTask(upon executor: Executor, start startNextTask: @escaping(Error) -> Task<SuccessValue>) -> Task<SuccessValue> {
```

Comment:
Totally not in this PR, but since I saw it while looking at this PR - there's a typo in the recover(upon:substituting:) variant here:

> The resulting task is cancellable in the same way the **recieving** task is.